go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/3scale/3scale-operator v0.5.0
 	github.com/Apicurio/apicurio-registry-operator v0.0.0-20200604085617-be91f2b38134
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Out of curiosity, why was this added? I can't seem to see it being used in this PR :thinking:

Reply:
This is added by `make vendor/fix` I've tried removing it and doing another `make vendor/fix` but it is still added. Interesting. Not sure why it would get added here and not in an earlier pr seeing as the package is in use in the code base already.